fmriprep/interfaces/confounds.py

```diff
@@ -163,10 +171,10 @@ def _run_interface(self, runtime):
         a_orig = a_comp_cor["component"]
         a_new = [f"a_comp_cor_{i:02d}" for i in range(len(a_orig))]
 
-        (components.rename(columns=dict(zip(c_orig, c_new)))
-                   .rename(columns=dict(zip(w_orig, w_new)))
-                   .rename(columns=dict(zip(a_orig, a_new)))
-         ).to_csv(self._results["components_file"], sep='\t', index=False)
+    final_components = components.rename(columns=dict(zip(c_orig, c_new)))
+    final_components.rename(columns=dict(zip(w_orig, w_new)), inplace=True)
+    final_components.rename(columns=dict(zip(a_orig, a_new)), inplace=True)
+    final_components.to_csv(self._results["components_file"], sep='\t', index=False)
```

Comment:
Sorry, looks like I gave you a dedented suggestion.

```suggestion
        final_components = components.rename(columns=dict(zip(c_orig, c_new)))
        final_components.rename(columns=dict(zip(w_orig, w_new)), inplace=True)
        final_components.rename(columns=dict(zip(a_orig, a_new)), inplace=True)
        final_components.to_csv(self._results["components_file"], sep='\t', index=False)
```